docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -645,6 +645,91 @@ See the following section to see if/how to apply the
+quantity of staple foods consumed as a proportion of total consumption 
+increases linearly from six months to two years of age, at which point it 
+reaches its peak and then remains constant.
+
+Further, we assumed that the full effect of the iron fortification 
+intervention takes six months to achieve and that the effect scales up in a 
+linear fashion between the onset of exposure and six months post exposure (
+this is likely a conservative assumption and the shape likely follows more of 
+a logarithmic curve). Additionally, we assumed that all individuals covered by 
+baseline coverage of iron fortification have been covered for at least six 
+months and therefore have already achieved the full effect of the intervention.
+
+.. todo::
+
+	Add justification and references to support for these assumptions.
+
+Therefore, the effect size of iron fortification on a simulant's hemoglobin 
+level **in the baseline scenario** should be determined as follows:
+
+.. code-block:: Python
+	
+	# at the draw level
+	MD = hb_md_distribution.rvs() # mean difference in hemoglobin concentration due to iron fortification
+		# note, hb_md_distribution defined above in the effect size section
+
+	# at the individual simulant level
+	if age_i < 0.5:
+		md_i = 0
+	elif age_i < 2:
+		md_i = MD * (age_i - 0.5) / 1.5
+	else: 
+		md_i = MD
+
+The effect size of the iron fortification on a simulant's hemoglobin level for 
+new **intervention** coverage of iron fortification should be determined as 
+follows: 
+
+.. code-block:: Python
+	
+	# MD = hb_md_distribution.rvs() : full effect size at the draw level 
+		# (mean difference in hemoglobin concentration due to iron fortification)
+	# md_i : effect size for an individual simulant at a given time-step, 
+		# dependent on age and time since coverage
+	# age_i : simulant age in years
+	# age_of_coverage_i : age at which simulant gained coverage access in years
+	# time_since_coverage_i : age_i - age_of_coverage_i
+	# age_of_coverage_i / 1.5 : fraction of full effect that will be achieved 
+		# in six months since onset of coverage access, dependent on 
+		# age_of_coverage_i [(age_of_coverage - 0.5 + 0.5) / (2 - 0.5)]
+	# time_since_coverage_i / 0.5 : fraction of effect that will be achieved 
+		# at six months post onset of coverage access, dependent on 
+		# time_since_coverage_i
+	# (age_i - 0.5) / 1.5 : fraction of full effect, dependent on age
+
+	# at the individual simulant level
+	if age_i < 0.5:
+		md_i = 0
+	if age_i > 0.5 and age_of_coverage < 1.5 and time_since_coverage_i < 0.5: 
+		md_i = MD * age_of_coverage_i / 1.5 * time_since_coverage_i / 0.5
+	if age_i > 0.5 and age_of_coverage < 1.5 and time_since_coverage_i >= 0.5: 
+		md_i = MD * (age_i - 0.5) / 1.5
+	if age_i > 0.5 and age_of_coverage >= 1.5 and time_since_coverage_i < 0.5: 
+		md_i = MD * time_since_coverage_i / 0.5
+	else:
+		md_i = MD
+
+See below for a visual representation:
+
+.. image:: iron_effect_scale_up.png
```

Comment:
Should we use .svg format instead of .png to make git tracking easier? I think matplotlib can save images in either format.

Reply:
Ah, I was just using a screenshot (which doesn't allow .svg as far as I could tell) because it formats easier than exporting, but I can do this if it's a priority.

Reply:
I'd say just try calling `plt.savefig('iron_effect_scale_up.svg')` and see how it comes out (note that you have to save the figure before calling `plt.show()`). If it looks good, let's use .svg, otherwise we can keep the .png.

Also, I'd like to see your code for generating this plot, e.g. in vivarium_data_analysis when you have time.